Tests/RxAnimatedTests/RxAnimatedTests.swift

```diff
@@ -0,0 +1,15 @@
+import XCTest
```

Comment:
I think we can remove test files, because there are no tests for Linux platforms.

Reply:
All tests in general or just the Linux test case? I think adding tests in the future would improve the library.

Reply:
Not all, only those which are template based.